iosApp/iosApp/iOSApp.swift

```diff
@@ -16,6 +16,11 @@ class AppDelegate: NSObject, UIApplicationDelegate {
                                           showAlertAfterCurrentVersionHasBeenReleasedForDays: 0)
         siren.wail()
 
+        let launchArguments = ProcessInfo.processInfo.arguments
+
+        if launchArguments.contains("--skipOnboarding") {
+            UserDefaults.standard.set(3, forKey: "notification_popup")
+        }
```

Comment:
I think this should have a comment explaining what is going on.

Reply:
Adding a description